test/server/views/e2e/_head.html.ejs

```diff
@@ -9,6 +9,7 @@
       }
       .test-bed {
         background: white;
+        width: 330px;
```

Comment:
I locked down the width here to prevent small viewport drift from device to device causing large visual regression diffs. This is an issue particularly in Galaxy phones.